src/components/members.component.js

```diff
@@ -65,10 +71,14 @@ export const MembersList = ({
       showsHorizontalScrollIndicator={false}
       renderItem={({ item }) =>
         <TouchableHighlight
-          onPress={() =>
-            navigation.navigate('Profile', {
-              user: item,
-            })}
+          onPress={() => {
+            navigation.navigate(
+              authUser.login === item.login ? 'AuthProfile' : 'Profile',
```

Comment:
I'm okay with leaving the logic here, I don't think this is a helper we would essentially need in `utils`.

Regarding @RolfKoenders's mention, you're right but this is something `prettier` and our precommit hook should take care of actually. @lex111 if I'm not mistaken this is going through the hook and getting prettified correct?